Kickstarter-iOS/Views/Cells/SettingsCurrencyCell.swift

```diff
@@ -3,17 +3,19 @@ import Library
 import Prelude
 
 final class SettingsCurrencyCell: UITableViewCell, NibLoading, ValueCell {
-  @IBOutlet fileprivate weak var titleLabel: UILabel!
+  @IBOutlet fileprivate weak var chevronImageView: UIImageView!
   @IBOutlet fileprivate weak var currentCurrencyLabel: UILabel!
   @IBOutlet fileprivate weak var separatorView: UIView!
+  @IBOutlet fileprivate weak var stackView: UIStackView!
+  @IBOutlet fileprivate weak var titleLabel: UILabel!
 
   private let viewModel: SettingsCurrencyCellViewModelType = SettingsCurrencyCellViewModel()
 
   func configureWith(value: SettingsCurrencyCellValue) {
     self.viewModel.inputs.configure(with: value)
 
     _ = titleLabel
-      |> UILabel.lens.text .~ value.cellType.title
+      |> \.text .~ value.cellType.title
```

Comment:
Wanna set the `backgroundColor` to `.white` to prevent rendering as blended layer?

Reply:
This cell has since been removed, let's do these changes as a sweep or as the merging of #573